src/content/blog/astro-520.mdx

```diff
@@ -40,9 +41,11 @@ yarn upgrade astro --latest
 
 ## Tailwind 4 support
 
-...
+Tailwind CSS now offers a `@tailwindcss/vite` plugin which is the preferred way to use Tailwind 4 in Astro, so we decided to deprecate the `@astrojs/tailwind` integration.
+
+If you want to upgrade, please uninstall the integration and either follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro), or use the updated `astro add tailwind` command if you're on 5.2. It will add the Vite plugin to your Astro config and create the default css file from now on.
```

Comment:
```suggestion
Tailwind CSS now offers a `@tailwindcss/vite` plugin which is the preferred way to use Tailwind 4 in Astro. Astro 5.2 adds native support for the Vite plugin, and `astro add tailwind` command will now add the plugin to your Astro config and create the default CSS file.

The `@astrojs/tailwind` integration is now deprecated, but will continue to work for older versions of Tailwind. If you want to upgrade, please uninstall the integration and either follow the [Tailwind documentation for manual installation](https://tailwindcss.com/docs/installation/framework-guides/astro), or use the updated `astro add tailwind` command.
```
I think it would be good to emphasis the new support, rather than the deprecation.